python/kiss_icp/datasets/kitti.py

```diff
@@ -34,6 +34,7 @@ def __init__(self, data_dir, sequence: int, *_, **__):
 
         self.scan_files = sorted(glob.glob(self.velodyne_dir + "*.bin"))
         self.calibration = self.read_calib_file(os.path.join(self.kitti_sequence_dir, "calib.txt"))
+        self.timestamps = self.read_timestamps(os.path.join(self.kitti_sequence_dir, "times.txt"))
```

Comment:
Why we need `read_timestamps` at all? I'd just put the `np.loadtxt` command here and remove the other function (unless the pipeline API asks for it)